Sources/ArcGISToolkit/Components/OfflineMapAreasView/PreplannedMapModel.swift

```diff
@@ -49,6 +94,20 @@ class PreplannedMapModel: ObservableObject, Identifiable {
         }
     }
     
+    /// Sets the model download preplanned offline map job if the job is in progress.
+    private func setDownloadJob() {
+        guard let preplannedMapAreaID else { return }
+        
+        for job in JobManager.shared.jobs {
```

Comment:
I saw you used the syntax in `MapViewModel`, up to you 

`for case let preplannedJob as DownloadPreplannedOfflineMapJob in …`